server/package.json

```diff
@@ -15,7 +15,9 @@
     "cors": "^2.8.5",
     "dotenv": "^16.0.3",
     "express": "^4.18.2",
+    "express-validator": "^6.14.2",
```

Comment:
You can choose whether to uninstall `express-validator`, I don't find it very critical since it'll be merged with my branch anyways